ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,187 @@
+
+name: Monorepo labels
+description: Import on 2023-05-09
+
+codes:
+  - code: A
+    name: Action 
+    description: used with GHA and trigger a certain process
+  - code: B
+    name: Dependencies 
+    description: Upgrade dependencies
+  - code: C
+    name: Contribution 
+    description: Upgrade dependencies
+  - code: D
+    name: Auditing 
+    description: Required when there were changes in the paths [CI/CD INPUT NEW FILE PATHS]. Also used to indicate the need for a documentation audit of DevRel.
```

Comment:
You could think about something like this https://github.com/paritytech/ci_cd/issues/719 to have the CI remind you :stuck_out_tongue:

Reply:
@alvicsam could we have the CI to add PR that touch files in a certain path automatically assign them to this project: https://github.com/orgs/paritytech/projects/103/views/1 ?

Reply:
Sure, I need a list of files/paths